### PR TITLE
Update VLC media player

### PR DESCRIPTION
**Pull request by fbernhart**
vlcmediaplayer

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/vlcmediaplayer.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>VLC media player icon</title><path d="M11.891 13.4624c3.7568 0 4.8538-1.347 5.0648-1.6739l-1.36-4.3338c-.36.34-1.3269.937-3.6798.937-2.2919 0-3.1879-.567-3.5199-.91l-1.3899 4.4218c.228.361 1.255 1.56 4.8848 1.56zm.026-9.6556a6.1427 6.1427 0 002.3759-.507L13.4349.566a.781.781 0 00-.406-.437S12.914 0 12.032 0c-.882 0-1.055.136-1.055.136a.733.733 0 00-.41.43l-.8839 2.8149a6.7697 6.7697 0 002.2339.426zM22.5896 23.318a.395.395 0 00-.016-.16l-2.334-7.6316a.647.647 0 00-.583-.431h-1.6629l.2.64c-.24.485-1.4879 2.3798-6.3027 2.3798-4.5998 0-5.8278-1.7299-6.1097-2.3079l.223-.712h-1.661a.647.647 0 00-.583.431L1.4264 23.159a.398.398 0 00-.017.15l-.002.02a1.843 1.843 0 00.024.226.528.528 0 00.51.445h20.1182a.528.528 0 00.51-.445 1.843 1.843 0 00.023-.226z"/></svg>
```
Comment (adamrusted) [inline icons/vlcmediaplayer.svg:1]: LGTM!
Could you get it down to 3 decimals though please? 😂

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/vlcmediaplayer.svg (hunk 1 above)
Comment (fbernhart) [reply icons/vlcmediaplayer.svg]: @adamrusted Fixed

Comment (fbernhart) [reply icons/vlcmediaplayer.svg]: I couldn't make it work with a precision of 3, even after rebuilding from source. I had to use 4 instead.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @fbernhart 🚀

**Pull request merged**